[EI-100] Docs review (#7199)
pantheon-systems/documentation: source/content/guides/edge-integrations/01-introduction.md

	- You can improve productivity by using existing content, style guides, media, and CMS integrations. No need to manage assets in multiple places.

- Performance-forward

> 	- Improve credibility with a fast, seamless customer experience, and distribute personalized content across dozens of global and US points of presence.

Reviewer: No comma before "and" because you are not listing three or more things, you're connecting two separate concepts.
```suggestion
	- Improve credibility with a fast, seamless customer experience and distribute personalized content across dozens of global and US points of presence.
```

- Cost-effective
	- Increase business impact by instrumenting and measuring success with your current analytics products. No need for new segmentation tooling.

- Geographic targeting

> 	- Based on the location of the visitor, the site can deliver a different homepage — e.g. Canadian visitors might see poutine, where US visitors would see pizza.

Reviewer: I think it might be easier to read if we just used "for example" rather than `e.g.`. Also, I think "whereas" makes more sense grammatically than "where" in this case.

```suggestion
	- Based on the location of the visitor, the site can deliver a different homepage — for example, Canadian visitors might see poutine, whereas US visitors would see pizza.
```

<dd>

> The cache layer stores and registers content variants utilizing the [vary header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Vary). The Vary HTTP response header describes the parts of the request message, aside from the method and URL, that influenced the content of the response it occurs in. It’s a key concept in the process of *content negotiation*. In HTTP, content negotiation is the mechanism that is used for serving different representations of a resource (WordPress Post, Drupal node) to the same URI to help the user agent specify which representation is best suited for the user (e.g. which document language, which personalization blocks, which content-encoding, which version of the content).

Reviewer: The parenthetical here `(WordPress Post, Drupal node)` is written as though those are the only options. I think it might communicate the intent better if we threw a "for example" in to show that these are just two examples. Originally, when this was written just for Drupal, I believe the "Page" that was in there was referring to Drupal pages, so there's at least two options in Drupal, and the WP post is not exclusive either, depending on how you've configured things.

```suggestion
The cache layer stores and registers content variants utilizing the [vary header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Vary). The Vary HTTP response header describes the parts of the request message, aside from the method and URL, that influenced the content of the response it occurs in. It’s a key concept in the process of *content negotiation*. In HTTP, content negotiation is the mechanism that is used for serving different representations of a resource (for example, a WordPress Post or Drupal node) to the same URI to help the user agent specify which representation is best suited for the user (e.g. which document language, which personalization blocks, which content-encoding, which version of the content).
```
